tests/ssg_test_suite/common.py

```diff
@@ -21,7 +21,11 @@
 Rule = namedtuple(
     "Rule", ["directory", "id", "short_id", "files"])
 
-_BENCHMARK_DIR = os.path.abspath(os.path.join(os.path.dirname(__file__), '../../linux_os/guide'))
+_BENCHMARK_DIRS = [
+        os.path.abspath(os.path.join(os.path.dirname(__file__), '../../linux_os/guide')),
+        os.path.abspath(os.path.join(os.path.dirname(__file__), '../../applications')),
+        ]
+
```

Comment:
Are changes on this file intended to be here? They are not mentioned in the PR description.
It adds possibility to run test scenarios for rules in `applications` benchmark.

Reply:
Yes, I have updated the description.